notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        MODEL_DISPLAY_NAME = "pet-adoption-prediction-model"
</pre><p>Please use UUID as part of the name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>done.</p>